test/logging/logging.go

```diff
@@ -67,7 +67,7 @@ func (e *zapMetricExporter) ExportView(vd *view.Data) {
 func (e *zapMetricExporter) ExportSpan(vd *trace.SpanData) {
 	duration := vd.EndTime.Sub(vd.StartTime)
 	// We will start the log entry with `metric` to identify it as a metric for parsing
-	e.logger.Infof("metric %s %d %d %s", vd.Name, vd.StartTime.UnixNano(), vd.EndTime.UnixNano(), duration)
+	e.logger.Debugf("metric %s %d %d %s", vd.Name, vd.StartTime.UnixNano(), vd.EndTime.UnixNano(), duration)
```

Comment:
This affects our metric aggregation jobs. What's the reason for the change?

Reply:
The reason was because for Zipkin tracing we are enabling Tracing for every request, which results in too many of these log statements especially for tests that use Poll(..) and E2E tests that make concurrent requests. Is it possible to change the metric aggregation jobs to look for Debug logs and enable verbose logging for those jobs, if not I can revert this change.

Reply:
I have reverted this back Infof. Enabling tracing for every request is required by Zipkin. But moving to Debug does affect other jobs, it makes sense to leave it as Infof.